lib/iris/tests/unit/fileformats/pyke_rules/compiled_krb/fc_rules_cf_fc/test_build_auxiliary_coordinate.py

```diff
@@ -16,7 +16,7 @@
 # along with Iris.  If not, see <http://www.gnu.org/licenses/>.
 """
 Test function :func:`iris.fileformats._pyke_rules.compiled_krb.\
-fc_rules_cf_fc.build_auxilliary_coordinate`.
+fc_rules_cf_fc.build_dimension_coordinate`.
```

Comment:
This change is coming from 604c1f366c1bd2ce0d55c6e8b8727ea57c9085af, but if you look at e2d856980ee8650fbd723c485accea77af53e4df (the corresponding commit from #1291) this change is not there! Looks like something went wrong with the extraction from #1291.